### PR TITLE
[API] Use swipe actions instead of deprecated edit actions

### PR DESCRIPTION
**Pull request by absidue**

### CODE REVIEW

File: Zebra/Tabs/Sources/Controllers/ZBSourceListTableViewController.m [hunk 1]
```diff
@@ -216,10 +219,12 @@ - (BOOL)tableView:(UITableView *)tableView shouldShowMenuForRowAtIndexPath:(NSIn
     return YES;
 }
 
+// FIXME: This is deprecated; move to swipe action?
 - (BOOL)tableView:(UITableView *)tableView canPerformAction:(SEL)action forRowAtIndexPath:(NSIndexPath *)indexPath withSender:(id)sender {
```
Comment (PoomSmart) [inline Zebra/Tabs/Sources/Controllers/ZBSourceListTableViewController.m:223]: Maybe you can change this in favour of `- (UISwipeActionsConfiguration *)tableView:(UITableView *)tableView 
leadingSwipeActionsConfigurationForRowAtIndexPath:(NSIndexPath *)indexPath`

Comment (absidue) [reply Zebra/Tabs/Sources/Controllers/ZBSourceListTableViewController.m:223]: Will do thanks.

**Comment by wstyres** [comment]
Going to wait until 1.2~beta2 to implement this.

**Comment by absidue** [comment]
Okay

**Pull request merged**